homeassistant/components/synology_dsm/const.py

```diff
@@ -43,6 +46,7 @@
 DEFAULT_SCAN_INTERVAL = 15  # min
 DEFAULT_TIMEOUT = ClientTimeout(total=60, connect=15)
 DEFAULT_SNAPSHOT_QUALITY = SNAPSHOT_PROFILE_BALANCED
+DEFAULT_BACKUP_PATH = "ha_backup"
```

Comment:
If they have multiple ha instances, should we call it `ha_backup_{slugify(hass.config.location_name)}`

Might make restore a bit harder though

Reply:
tbh, didn't thought about multiple HA instances, only about multiple Synology-DSM instances :see_no_evil:

Reply:
further we store an `instance_id` in the backup meta data - not sure if this is already enough to distinguish the backups for the different HA instances?

Reply:
I was thinking users would want to be able to distinguish without having to dig through meta data as its unlikely they will know how